syntax = "proto3";

package aglcom;

service DemoMessage {
  rpc SendDemo (DemoRequest) returns (DemoReply) {}
}

message DemoRequest {
  string name = 1;
}

message DemoReply {
  string message = 1;
}
